controllers/bookingsController.js

```diff
@@ -52,12 +54,426 @@ class BookingsController extends BaseController {
     const { eventId } = req.params;
     const { quantity_bought } = req.body;
     const { user_id } = req.body;
+    console.log(req?.body?.email);
+    let transporter = nodemailer.createTransport({
+      host: "smtp.gmail.com",
+      port: 587,
+      service: "gmail",
+      secure: false, // Use `true` for port 465, `false` for all other ports
+      auth: {
+        user: "eventlink.ptbc9@gmail.com",
+        pass: "jhsu kyho asqw jtur",
```

Comment:
pls use env for the password

Reply:
How does the email notification works? 

If we want to send email when user has successfully booked and has successfully made payment, the nodemailer should be under async **getSessionStatus** or **insertOne** (this function is invoked when user has successfully made stripe payment and we are inputting their booking records into our database) and **insertOneFree** (for free events without stripe payment, this is also a function where we are inputting their booking records into our database)